content/docs/replenishment/r-and-r-forms.md

```diff
@@ -72,26 +72,28 @@ Once you are happy with your inputs, click `OK` to generate the form. You will b
+| **Adjustments +/-\***                 | Losses/adjustments made for this item during the period. Can be positive or negative. Populated by data from Stocktakes or Inventory Adjustments.                                                             |
+| **Stock out duration\***              | Number of days in the period where stock on hand for the item was 0.                                                                                                                                          |
+| **Final balance**                     | Stock on hand for the item at the end of the period. Calculation is <code>Initial balance + Received - Consumed + Adjustments</code>                                                                          |
+| **AMC/AMD**                           | Average monthly consumption (distribution) over the last 3 periods                                                                                                                                            |
+| **Maximum**                           | Ideal amount of stock to have on hand, requested quantity could be any amount up to this value. Calculated as <code>AMC x 2</code>                                                                            |
+| **Expiry\***                          | Expiry date of the earliest expiring available batch of this item                                                                                                                                             |
+| **Requested\***                       | Quantity to be requested in the requisition. Calculated as <code>Maximum - Final balance</code>                                                                                                               |
+| **Low stock**                         | Warning indicator if your final balance is low in comparison to the ideal stock level. Will show `!` when `Final balance` is less than half of `Maximum`, and `!!` when less than quarter                     |
+| **Comment\***                         | You can add any comments to the line as needed                                                                                                                                                                |
+| **Confirmed\***                       | Use this column to keep track of which lines are complete. Acts as the save button for changes to a line.                                                                                                     |
+| **Approved Quantity**                 | Once the R&R Form is finalised, this column will show the quantity approved by the supplier                                                                                                                   |
```

Comment:
strictly speaking, it isn't the supplier that approves in the current example - it's a gov dept which inserts itself into the process. it's a little clumsier but perhaps 
```suggestion
| **Approved Quantity**                 | Once the R&R Form is finalised and authorisation is configured, this column will show the quantity approved by the authoriser                  |
```